sunpy/timeseries/timeseriesbase.py

```diff
@@ -152,29 +140,31 @@ def columns(self):
         """
         A list of all the names of the columns in the data.
         """
-        return list(self._data.columns.values)
+        names = self._data.colnames
+        names.remove('time')
+        return names
 
     @property
     def index(self):
```

Comment:
more of a general comment for discussion - should we deprecate index? and have it as `ts.time` instead?

Reply:
👍 after reading your comment above I independently came to this conclusion